examples/vercel-edge-runtime/README.md

```diff
@@ -0,0 +1,6 @@
+- [Vercel Edge Runtime](https://edge-runtime.vercel.app)
+- Vanilla TRPCClient in node
+
+---
+
+Created by [tomlienard](https://github.com/tomlienard).
```

Comment:
I reckon that 99% of people use Next.js, maybe we should make an example with https://github.com/trpc/next-minimal-starter on the edge?

Reply:
Added a new next-edge-runtime example that is based on next-minimal-starter. It uses the `fetch` adapter but maybe we should add an option to `createNextApiHandler` in `@trpc/next` to make it easier for the users (e.g `edge: true`)